desktop-exporter/app/components/detail-view/span-field.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import {
+  Flex,
+  Box,
+  Tag,
+  Text,
+  TagLabel,
+  useColorModeValue,
+} from "@chakra-ui/react";
+
+type SpanFieldProps = {
+  fieldName: string;
+  fieldValue: number | string | boolean | null;
+};
+
+export function SpanField(props: SpanFieldProps) {
+  let { fieldName, fieldValue } = props;
+  let fieldNameColour = useColorModeValue("gray.600", "gray.400");
+
+  if (fieldValue == null || fieldValue === "") {
+    return null;
+  }
```

Comment:
We probably want to render them in this case. If you forget to set a value for a field and it's the empty string, it would likely be pretty confusing to see it not present and make figuring out your mistake harder.